apps/institution-service/src/app.ts

```diff
@@ -30,6 +33,16 @@ const cacheListLimiter = createLimiter({
   requestLimit: 3,
 });
 
+sequelize
+  .sync()
+  .then(() => {
+    defineAssociations();
```

Comment:
I ran into a really weird problem where certain defined associations were working while testing manually, but failing in the specs. Did some research and it seems like it's best practice to load the associations after all the models are initialized and before running tests or the server.